services/web-app/components/breadcrumb-nav/breadcrumb-nav.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Breadcrumb } from '@carbon/react'
+import clsx from 'clsx'
+import Link from 'next/link'
+
+import styles from './breadcrumb-nav.module.scss'
+
+const BreadcrumbNav = ({ libraryId, contentId, title, className }) => {
+  return (
+    <Breadcrumb className={clsx(styles.container, className)}>
```

Comment:
Do we want to use Carbon's `Breadcrumb` component but not also the `BreadcrumbItem` components that are meant to go inside of it? If not using `BreadcrumbItem`, what purpose does the `Breadcrumb` serve?